packages/libs/error-stack/src/lib.rs

```diff
@@ -477,6 +477,8 @@ pub mod fmt;
 mod fmt;
 #[cfg(feature = "std")]
 mod hook;
+#[cfg(feature = "serde")]
+mod ser;
```

Comment:
Nit: Do we want to rename the module to `serde`? I don't feel strongly, just want to flag.

Reply:
fixed in [0147679](https://github.com/hashintel/hash/pull/1290/commits/014767941aa81eb87e56bdfc21492f85ee8eae06)